### ISSUE TITLE
Emails search box - add possibility search the text in body of email

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#3342]
#### Issue

It is possible only search the text in those fields: FROM, TO, SUBJECT

Many times the information is in body of emails... we are using the email clients for search the informations because we can't search the text in body of emails in SuiteCRM

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Emails search box - add possibility search the text in body of email](https://trello.com/c/FPgGqgtu/6-emails-search-box-add-possibility-search-the-text-in-body-of-email)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Emails search box - add possibility search the text in body of email](https://trello.com/c/FPgGqgtu/6-emails-search-box-add-possibility-search-the-text-in-body-of-email)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
